src/frontend/Komponenter/Behandling/Totrinnskontroll/ModalOpprettOgFerdigstilleOppgaver.tsx

```diff
@@ -46,19 +46,22 @@ export const ModalOpprettOgFerdigstilleOppgaver: FC<{
         );
 
     const kanVelgeMellomFlereOppgavetyper = (oppgavetyperSomKanOpprettes ?? []).length > 1;
+    const harValgtAnnetEnnInntektskontroll =
+        kanVelgeMellomFlereOppgavetyper &&
+        !oppgavetyperSomSkalOpprettes.includes(
+            OppgaveTypeForOpprettelse.INNTEKTSKONTROLL_SELVSTENDIG_NÆRINGSDRIVENDE
+        );
```

Comment:
`harValgtAnnetEnnInntektskontroll` Må du ikke egentlig sjekke at lista er større enn 1 OG at includes returnerer null? 

Eventuelt først filtrere lista for `INNTEKTSKONTROLL_SELVSTENDIG_NÆRINGSDRIVENDE` og deretter sjekke at størrelsen er større enn 0?